### ISSUE TITLE
MenuButton: missing support for aria-label

### ISSUE THREAD

**Issue by b0rengar** [primefaces/primefaces#3441]
## 1) Environment
- PrimeFaces version: 6.2
- Does it work on the newest released PrimeFaces version? Version? No
- Does it work on the newest sources in GitHub? (Build by source -> https://github.com/primefaces/primefaces/wiki/Building-From-Source) not tested
- Application server + version: not relevant
- Affected browsers: not relevant

## 2) Expected behavior
MenuButton tag has an attribute to set the aria-label (for icon-only buttons)

## 3) Actual behavior
there is no way to set the aria-label on MenuButton

**Comment by cnsgithub** [comment]
PR: https://github.com/primefaces/primefaces/pull/3452
